android-studio

```diff
@@ -34,10 +34,14 @@
     "autoupdate": {
         "architecture": {
             "64bit": {
-                "url": "https://dl.google.com/dl/android/studio/ide-zips/$version/android-studio-ide-$matchBuild-windows.zip"            },
+                "url": "https://dl.google.com/dl/android/studio/ide-zips/$version/android-studio-ide-$matchBuild-windows.zip"
+            },
             "32bit": {
                 "url": "https://dl.google.com/dl/android/studio/ide-zips/$version/android-studio-ide-$matchBuild-windows32.zip"
             }
+        },
+        "hash": {
+            "url": "https://developer.android.com/studio/"
```

Comment:
🥇  I found that only when I download the source with curl.
`view-source:https://developer.android.com/studio/index.html` not showing that